skeleton/frontend/src/Frontend.hs

```diff
@@ -4,8 +4,11 @@
 
 module Frontend where
 
+import Control.Monad
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as T
+import GHCJS.DOM.Types (liftJSM)
```

Comment:
I expect these will require dependencies to `frontend.cabal`. CI should confirm :smile: